examples/connect/off-host-custom-images/README.md

```diff
@@ -0,0 +1,24 @@
+## Off-Host Execution with Custom Images
```

Comment:
This is a fantastic question that I do not believe has been answered yet. I lean towards "here for this one" because it includes mostly code - a collection of yaml files. If there was more prose (not my strong suit), it might make more sense there, but we also still need more deliberation on helm and where it lands on the docs site anyways haha. (I linked a new issue for next steps there)